src/Stratis.Bitcoin.Features.Wallet.Tests/WalletControllerTest.cs

```diff
@@ -761,7 +759,7 @@ public void GetGeneralInfoSuccessfullyReturnsWalletGeneralInfoModel()
             Assert.Equal(0, resultValue.ConnectedNodes);
             Assert.Equal(tip.Height, resultValue.ChainTip);
             Assert.True(resultValue.IsDecrypted);
-            Assert.Equal(testWalletPath, resultValue.WalletFilePath);
+            //Assert.Equal(testWalletPath, resultValue.WalletFilePath);
```

Comment:
Please remove or add comment on why this commented out.

Reply:
Restored.